pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/errorprone/BeanMembersShouldSerializeRule.java

```diff
@@ -74,20 +74,20 @@ public Object visit(ASTClassOrInterfaceDeclaration node, Object data) {
 
         Map<VariableNameDeclaration, List<NameOccurrence>> vars = node.getScope()
                 .getDeclarations(VariableNameDeclaration.class);
-        for (VariableNameDeclaration decl : vars.keySet()) {
-            AccessNode accessNodeParent = decl.getAccessNodeParent();
-            if (vars.get(decl).isEmpty() || accessNodeParent.isTransient() || accessNodeParent.isStatic()) {
+        for (Map.Entry<VariableNameDeclaration, List<NameOccurrence>> entry : vars.entrySet()) {
```

Comment:
I'd add a `VariableNameDeclaration decl = entry.getKey();` here for readability

Reply:
Fixed